README.md

```diff
@@ -19,24 +19,41 @@ In order to access the development branch on contentful during development set t
 ENV=dev yarn develop
 ```
 
+and your .env file to match the contentful keys and tokens
```

Comment:
Might be nice to add this instruction in before [line 13, when you're told to run `yarn develop`](https://github.com/yldio/yld.io/pull/165/files#diff-04c6e90faac2675aa89e2176d2eec7d8L13)?

We can also probably delete lines 16-20 altogether, since the development branch of Contentful isn't being kept up to date or used at all really.